Frends.MongoDB.Delete/Frends.MongoDB.Delete.Tests/UnitTests.cs

```diff
@@ -119,7 +119,7 @@ public async Task Test_Delete_Array_DeleteMany()
         };
```

Comment:
https://github.com/FrendsPlatform/Frends.MongoDB/blob/c79971d87b26eed3cd98c01ede770d8832c2d4c0/Frends.MongoDB.Delete/Frends.MongoDB.Delete.Tests/UnitTests.cs#L52
Please split this into their own asserts.

https://github.com/FrendsPlatform/Frends.MongoDB/blob/c79971d87b26eed3cd98c01ede770d8832c2d4c0/Frends.MongoDB.Delete/Frends.MongoDB.Delete.Tests/UnitTests.cs#L68
Same here.

https://github.com/FrendsPlatform/Frends.MongoDB/blob/c79971d87b26eed3cd98c01ede770d8832c2d4c0/Frends.MongoDB.Delete/Frends.MongoDB.Delete.Tests/UnitTests.cs#L84
Same here.

https://github.com/FrendsPlatform/Frends.MongoDB/blob/c79971d87b26eed3cd98c01ede770d8832c2d4c0/Frends.MongoDB.Delete/Frends.MongoDB.Delete.Tests/UnitTests.cs#L103
Same here.